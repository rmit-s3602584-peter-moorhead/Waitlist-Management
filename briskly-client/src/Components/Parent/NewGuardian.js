import React, {Component} from 'react'
import AddressContact from "./AddressContact.js"
import Occupation from "./Occupation.js"

class NewGuardian extends Component{

    /**
     * 
     * This component is used to add a new part for the form which allows
     * the parent to add a second guardian
     */

    //the constructor is used to initialize imoprtant variabes here
    constructor(props){
        super(props)
        //to know if to show the file input or not
        this.showFileInputOptionClicked = false
        //all possible relationship status for the parent/guardian
        this.relationshipStatus = ['Single', 'Married', 'Divorced', 'Widowed']

    }
    //this event is called when the "Remove second parent/guardian" button is clicked
    delete = (e) => {
        //it removes this component from the parent component.
        this.props.delete()
    }
    //this event is called when the user answers YES for the custody question (Last question)
    showFileInput = () => {
        //the file input will appear
        this.showFileInputOptionClicked = true
        this.forceUpdate()
    }
    //this event is called when the user answers NO for the custody question (Last question)
    hideFileInput = () =>{
        //the file input WILL GO AWAY/DISAPPEAR
        this.showFileInputOptionClicked = false
        this.forceUpdate()
    }

    fileInput(){
        //this method is responsibe for returning the HTML tags that are responsible for showing
        //the file input when the user answeres YES
        if(this.showFileInputOptionClicked){
            return <div style={{marginTop: '3%', display: 'flex', flexDirection: 'row'}}>
            <div style={{width: '30%'}}><label style={{marginRight: '%}'}}>Provide a copy of the agreement</label></div><div><input name="agreementFile" type="file"/></div>
            </div>
        }else{
            //returns null otherwise
            return null
        }
    }

    render(){
        return(

<div id="newGuardian">
<button id="removeGuardian" class="btn btn-primary" type="button" style={{backgroundColor: 'rgb(184, 5, 5)'}} onClick={this.delete.bind(this)}>Remove second parent/guardian</button>
    <div><label class="title" style={{marginTop: '2%'}}>Second Parent/Guardian</label></div>
    
            <div class="form-group">
            <div style={{display: 'flex',flexDirection: 'row'}}>
                <div style={{width: '13.5%',alignSelf: 'center'}}><label style={{margin: '0px'}}>First Name:</label></div>
                <div style={{width: '70%'}}><input class="form-control" type="text" name="firstName2" placeholder="First Name" style={{width: '70%',padding: '0px',marginLeft: '1%'}}/></div>
            </div>
        </div>
        <div class="form-group">
            <div style={{display: 'flex',flexDirection: 'row'}}>
                <div style={{width: '13.5%',alignSelf: 'center'}}><label style={{margin: '0px'}}>Last Name:</label></div>
                <div style={{width: '70%'}}><input class="form-control" type="text" name="lastName2" placeholder="Last Name" style={{width: '70%',padding: '0px',marginLeft: '1%'}}/></div>
            </div>
        </div>
        <div class="form-group">
            <div style={{display: 'flex',flexDirection: 'row'}}>
                <div style={{width: '13.5%',alignSelf: 'center'}}><label style={{margin: '0px'}}>Username:</label></div>
                <div style={{width: '70%'}}><input class="form-control" type="text" name="username2" placeholder="Username" style={{width: '70%',padding: '0px',marginLeft: '1%'}}/></div>
            </div>
        </div>
        <div class="form-group">
                    <div style={{display: 'flex',flexDirection: 'row'}}>
                        <div style={{width:'14%', marginTop: '1%'}}><label style={{margin: '0px'}}>Relationship:</label></div>
                        <div style={{width: "25%"}}>
                            <select name="relationship2" class="form-control" >
                                <option>Select Relationship</option>
                                {
                                    //displaying the relationships from the array in the constructor
                                    this.relationshipStatus.map((item, i) => <option name="relationship2" value={this.relationshipStatus[i]}>{this.relationshipStatus[i]}</option>)
                                }

                        </select>
                    </div>
                </div>
        </div>
<div class="form-group">
    <div style={{display: 'flex',flexDirection: 'row'}}>
        <div style={{width: '14%'}}><label style={{margin: '0px',marginTop: '5%'}}>Date of birth:</label></div>
        <div ><input class="form-control" name="birthday2" type="date" /></div>
    </div>
</div>

<AddressContact keyToUse={0} country="country2" state="state2" city="city2" title={"Address and Contact information"} 
    names={["homeAddress2", "homeZipCode2", "countryCode2", "phoneNumber2", "personalEmail2"]} shortCode={this.props.shortCode}/>


        
        
        
        
       <Occupation names={["occupation2", "company2", "natureOfBusiness2", "yearsWithCompany2"]}/>
        

<AddressContact keyToUse={1} country="businessCountry2" state="businessState2" city="businessCity2" title={"Business Address"}
names={["businessAddress2", "businessZipCode2", "businessCountryCode2", "businessPhoneNumber2", "businessEmail2"]} shortCode={this.props.shortCode}/>





            <div><label class="title" style={{marginTop: '3%'}}>Parental Situation</label>
        <div style={{display: 'flex',flexDirection: 'row'}}>
            <div style={{width: '30%'}}><label>Child lives with</label></div>
            <div>
                <div class="form-check"><input class="form-check-input" name="Child-Lives-With" value="Both" type="radio" id="formCheck-1"/><label class="form-check-label" for="formCheck-1">Both parents/guardians</label></div>
                <div class="form-check"><input class="form-check-input" name="Child-Lives-With" value="One" type="radio" id="formCheck-2"/><label class="form-check-label" for="formCheck-2">One parent/guardian</label></div>
            </div>
        </div>
    </div>
    <div style={{marginTop: '3%'}}>
        <div style={{display: 'flex',flexDirection: 'row'}}>
            <div style={{width: '30%'}}><label>Is there a custody agreement?</label></div>
            <div style={{width: '20%'}}>
                <div class="form-check"><input class="form-check-input" name="Custody-Agreement" onClick={this.showFileInput} value="1" type="radio" id="formCheck-3"/><label class="form-check-label" for="formCheck-3">Yes</label></div>
                <div class="form-check"><input class="form-check-input" name="Custody-Agreement" onClick={this.hideFileInput} value="0" type="radio" id="formCheck-4"/><label class="form-check-label" for="formCheck-4">No</label></div>
            </div>
        </div>
        {
            //once the showFileInputOptionClicked changes to true then the below function will return
            //the HTML tags for showing the file input.
            this.fileInput()
        }
    </div>
</div>

        )
    }



}

export default NewGuardian