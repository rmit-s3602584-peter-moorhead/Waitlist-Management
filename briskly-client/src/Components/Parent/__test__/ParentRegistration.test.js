import React from 'react'
import ReactDOM from 'react-dom'
import ParentRegistration from '../ParentRegistration.js'
import NewGuardian from '../NewGuardian.js'
import "../../../setupTests"
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import renderer from "react-test-renderer"

import { shallow, mount } from 'enzyme';
import ReactTestUtils, { act } from 'react-dom/test-utils';
import { wrap } from 'module'

afterEach(cleanup)


it("renders without crashing", () =>{
    //testing it the page actually renders or not
    const div = document.createElement("div");
    ReactDOM.render(<ParentRegistration></ParentRegistration>, div)
})


it("matches snapshot", () => {
    //creating a snapshot test
    const tree = renderer.create(<ParentRegistration></ParentRegistration>).toJSON()
    expect(tree).toMatchSnapshot()
})

it("testing to see if the second parent/guardian appears when the page is first loaded or not", () => {
    //this test tests to see if the form to add another guaridan appears or not
    const wrapper = mount(<ParentRegistration/>)
    const newGuardian = wrapper.find('#newGuardian')
    expect(newGuardian).toHaveLength(0)
})


it("testing the Add Another Guardian button", () => {
    //this test tests the ability to add another parent/guardian by clicking the button
    const wrapper = mount(<ParentRegistration/>)
    const btn = wrapper.find('#addGuardian')
    btn.simulate('click')
    const newGuardian = wrapper.find('#newGuardian')
    expect(newGuardian).toHaveLength(1)

})

it("testing the remove second parent/guardian button", () => {
    //this test tests to see if the remove second parent/guardian appears or not
    const wrapper = mount(<ParentRegistration/>)
    const btn = wrapper.find('#addGuardian')
    btn.simulate('click')
    wrapper.update()
    const btn2 = wrapper.find('#removeGuardian')
    
    expect(btn2.text()).toBe("Remove second parent/guardian")
    expect(wrapper.find('#addGuardian')).toHaveLength(0)

})
it("testing the remove second parent/guardian button if it removes the add guardian button or not", () => {
    //this test tests to see if the add guardian button is removed or not
    const wrapper = mount(<ParentRegistration/>)
    const btn = wrapper.find('#addGuardian')
    btn.simulate('click')
    wrapper.update()
    expect(wrapper.find('#addGuardian')).toHaveLength(0)

})
//expect(component.find('#item-id').prop('style')).toHaveProperty('backgroundSize', '100%');
//free
//premium
it("testing to see if the button with the account features is pressed or not", () => {
    //this tests the selected button 
    const wrapper = mount(<ParentRegistration/>)
    const btn = wrapper.find('#free')
    btn.simulate('click')
    wrapper.update()
    // console.log(wrapper.find('#free').prop('style'))
    expect(wrapper.find('#free').prop('style')).toStrictEqual({
                borderStyle: 'solid',
                borderColor: 'rgb(6,84,113)',
                borderRadius: '2%',
                width: '40%',
                marginRight: '2.5%',
                marginLeft: '7.5%',
                backgroundColor: 'rgb(26, 101, 126)',
                color: "white"
    })
})
it("testing to see if the button with the account features is pressed or not", () => {
    //this tests the other button..not the selected one
    const wrapper = mount(<ParentRegistration/>)
    const btn = wrapper.find('#free')
    btn.simulate('click')
    wrapper.update()
    // console.log(wrapper.find('#free').prop('style'))
    expect(wrapper.find('#premium').prop('style')).toStrictEqual({
                borderStyle: 'solid',
                borderColor: 'rgb(6,84,113)',
                borderRadius: '2%',
                width: '40%',
                marginRight: '7.5%',
                marginLeft: '2.5%',
                backgroundColor: 'white'
    })
})