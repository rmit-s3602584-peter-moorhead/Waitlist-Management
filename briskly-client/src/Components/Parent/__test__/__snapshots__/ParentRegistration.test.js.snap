// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  class="register-photo"
>
  <div
    class="form-container"
  >
    <form
      method="post"
      onSubmit={[Function]}
    >
      <h2
        class="text-center"
      >
        Complete Profile
      </h2>
      <label
        class="title"
      >
        Personal Information
      </label>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "alignSelf": "center",
                "width": "13.5%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              First Name:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <input
              class="form-control"
              name="firstName1"
              placeholder="First Name"
              style={
                Object {
                  "marginLeft": "1%",
                  "padding": "0px",
                  "width": "70%",
                }
              }
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "alignSelf": "center",
                "width": "13.5%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              Last Name:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <input
              class="form-control"
              name="lastName1"
              placeholder="Last Name"
              style={
                Object {
                  "marginLeft": "1%",
                  "padding": "0px",
                  "width": "70%",
                }
              }
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "alignSelf": "center",
                "width": "13.5%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              Password:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <input
              class="form-control"
              name="userName1"
              placeholder="Password"
              style={
                Object {
                  "marginLeft": "1%",
                  "padding": "0px",
                  "width": "70%",
                }
              }
              type="password"
            />
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "alignSelf": "center",
                "width": "13.5%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              Resenter Password:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <input
              class="form-control"
              name="password"
              placeholder="Password"
              style={
                Object {
                  "marginLeft": "1%",
                  "padding": "0px",
                  "width": "70%",
                }
              }
              type="password"
            />
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "alignSelf": "center",
                "width": "13.5%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              Username:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <input
              class="form-control"
              name="passwordRepeat"
              placeholder="Username"
              style={
                Object {
                  "marginLeft": "1%",
                  "padding": "0px",
                  "width": "70%",
                }
              }
              type="text"
            />
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "marginTop": "1%",
                "width": "14%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                }
              }
            >
              Relationship:
            </label>
          </div>
          <div
            style={
              Object {
                "width": "25%",
              }
            }
          >
            <select
              class="form-control"
              name="relationship1"
            >
              <option>
                Select Relationship
              </option>
              <option
                name="relationship1"
                value="Single"
              >
                Single
              </option>
              <option
                name="relationship1"
                value="Married"
              >
                Married
              </option>
              <option
                name="relationship1"
                value="Divorced"
              >
                Divorced
              </option>
              <option
                name="relationship1"
                value="Widowed"
              >
                Widowed
              </option>
            </select>
          </div>
        </div>
      </div>
      <div
        class="form-group"
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
            }
          }
        >
          <div
            style={
              Object {
                "width": "14%",
              }
            }
          >
            <label
              style={
                Object {
                  "margin": "0px",
                  "marginTop": "5%",
                }
              }
            >
              Date of birth:
            </label>
          </div>
          <div>
            <input
              class="form-control"
              name="birthday1"
              type="date"
            />
          </div>
        </div>
      </div>
      <div>
        <label
          class="title"
        >
          Address and Contact information
        </label>
        <div
          class="form-group"
        >
          <label>
            Street Address:
          </label>
          <input
            class="form-control"
            name="homeAddress1"
            placeholder="Street Address"
            type="text"
          />
        </div>
        <div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                    }
                  }
                >
                  Country:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  id="country"
                  name="country1"
                  onChange={[Function]}
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select Country
                  </option>
                </select>
              </div>
            </div>
          </div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                    }
                  }
                >
                  State/Province:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  name="state1"
                  onChange={[Function]}
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select State/Province
                  </option>
                </select>
              </div>
            </div>
          </div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                      "margin": "0px",
                    }
                  }
                >
                  City:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  name="city1"
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select City
                  </option>
                </select>
              </div>
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Postal/Zip code:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="homeZipCode1"
                placeholder="Post/Zip code"
                style={
                  Object {
                    "width": "40%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Phone Number:
              </label>
            </div>
            <div
              class="dropdown dropDownMenu"
              style={
                Object {
                  "marginRight": "2%",
                  "width": "20%",
                }
              }
            >
              <select
                class="form-control countryStateCityDropdown"
                name="countryCode1"
                style={
                  Object {
                    "fontSize": "100%",
                    "height": "100%",
                    "width": "100%",
                  }
                }
              >
                <option>
                  Country Code
                </option>
              </select>
            </div>
            <input
              class="form-control"
              name="phoneNumber1"
              placeholder="XXXXXXXXX"
              style={
                Object {
                  "width": "25%",
                }
              }
              type="text"
            />
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Email Address:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="personalEmail1"
                placeholder="e.g. jon@gmail.com"
                style={
                  Object {
                    "width": "60%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
      <div>
        <label
          class="title"
        >
          Occupational Information
        </label>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Occupation/Position:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="occupation1"
                placeholder="e.g. jon@gmail.com"
                style={
                  Object {
                    "marginLeft": "1%",
                    "padding": "0px",
                    "width": "70%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Company:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="company1"
                placeholder="Company Name"
                style={
                  Object {
                    "marginLeft": "1%",
                    "padding": "0px",
                    "width": "70%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Nature of business:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="natureOfBusiness1"
                placeholder="IT/Marketing/Retail/Service"
                style={
                  Object {
                    "marginLeft": "1%",
                    "padding": "0px",
                    "width": "70%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Years with company:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="yearsWithCompany1"
                placeholder="e.g. 10"
                style={
                  Object {
                    "marginLeft": "1%",
                    "padding": "0px",
                    "width": "20%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
      <div>
        <label
          class="title"
        >
          Business Address
        </label>
        <div
          class="form-group"
        >
          <label>
            Street Address:
          </label>
          <input
            class="form-control"
            name="businessAddress1"
            placeholder="Street Address"
            type="text"
          />
        </div>
        <div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                    }
                  }
                >
                  Country:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  id="country"
                  name="businessCountry1"
                  onChange={[Function]}
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select Country
                  </option>
                </select>
              </div>
            </div>
          </div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                    }
                  }
                >
                  State/Province:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  name="businessState1"
                  onChange={[Function]}
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select State/Province
                  </option>
                </select>
              </div>
            </div>
          </div>
          <div
            class="form-group"
          >
            <div
              style={
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                }
              }
            >
              <div
                class="countryStateCityLabel"
              >
                <label
                  style={
                    Object {
                      "fontSize": "calc(0.5em + 1vmin)",
                      "margin": "0px",
                    }
                  }
                >
                  City:
                </label>
              </div>
              <div
                style={
                  Object {
                    "width": "50%",
                  }
                }
              >
                <select
                  class="form-control countryStateCityDropdown"
                  name="businessCity1"
                >
                  <option
                    aria-expanded="false"
                    data-toggle="dropdown"
                    style={
                      Object {
                        "backgroundColor": "rgb(219,222,225)",
                        "color": "rgb(0,0,0)",
                        "margin": 0,
                      }
                    }
                  >
                    Select City
                  </option>
                </select>
              </div>
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Postal/Zip code:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="businessZipCode1"
                placeholder="Post/Zip code"
                style={
                  Object {
                    "width": "40%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Phone Number:
              </label>
            </div>
            <div
              class="dropdown dropDownMenu"
              style={
                Object {
                  "marginRight": "2%",
                  "width": "20%",
                }
              }
            >
              <select
                class="form-control countryStateCityDropdown"
                name="businessCountryCode1"
                style={
                  Object {
                    "fontSize": "100%",
                    "height": "100%",
                    "width": "100%",
                  }
                }
              >
                <option>
                  Country Code
                </option>
              </select>
            </div>
            <input
              class="form-control"
              name="businessPhoneNumber1"
              placeholder="XXXXXXXXX"
              style={
                Object {
                  "width": "25%",
                }
              }
              type="text"
            />
          </div>
        </div>
        <div
          class="form-group"
        >
          <div
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <div
              style={
                Object {
                  "alignSelf": "center",
                  "width": "20%",
                }
              }
            >
              <label
                style={
                  Object {
                    "margin": "0px",
                  }
                }
              >
                Email Address:
              </label>
            </div>
            <div
              style={
                Object {
                  "width": "70%",
                }
              }
            >
              <input
                class="form-control"
                name="businessEmail1"
                placeholder="e.g. jon@gmail.com"
                style={
                  Object {
                    "width": "60%",
                  }
                }
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
      <div>
        <button
          class="btn btn-primary"
          id="addGuardian"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "rgb(6,84,113)",
            }
          }
          type="button"
        >
          Add Another Parent/Guardian
        </button>
      </div>
      <div>
        <label
          class="title"
          style={
            Object {
              "marginTop": "3%",
            }
          }
        >
          Select profile type
        </label>
        <div
          class="form-row justify-content-center features"
        >
          <button
            class="col-sm-6 col-md-5 col-lg-4 item ProfileType"
            id="free"
            onClick={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "backgroundColor": "white",
                "borderColor": "rgb(6,84,113)",
                "borderRadius": "2%",
                "borderStyle": "solid",
                "marginLeft": "7.5%",
                "marginRight": "2.5%",
                "width": "40%",
              }
            }
            type="button"
          >
            <span
              class="box"
              style={
                Object {
                  "paddingBottom": "50%",
                }
              }
            >
              <h3
                class="name"
              >
                Free
              </h3>
              <span
                class="box"
              >
                <span>
                  <label>
                    - Feature 1
                  </label>
                </span>
                <span>
                  <label>
                    - Feature 2
                  </label>
                </span>
                <span>
                  <label>
                    - Feature 3
                  </label>
                </span>
              </span>
            </span>
          </button>
          <button
            class="col-sm-6 col-md-5 col-lg-4 item ProfileType"
            id="premium"
            onClick={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "backgroundColor": "white",
                "borderColor": "rgb(6,84,113)",
                "borderRadius": "2%",
                "borderStyle": "solid",
                "marginLeft": "2.5%",
                "marginRight": "7.5%",
                "width": "40%",
              }
            }
            type="button"
          >
            <span
              class="box"
              style={
                Object {
                  "paddingBottom": "10%",
                }
              }
            >
              <h3
                class="name"
              >
                Premium
              </h3>
              <span>
                <label>
                  - Feature 1
                </label>
              </span>
              <span>
                <label>
                  - Feature 2
                </label>
              </span>
              <span>
                <label>
                  - Feature 3
                </label>
              </span>
              <span>
                <label>
                  - Feature 4
                </label>
              </span>
              <span>
                <label>
                  - Feature 5
                </label>
              </span>
              <span>
                <label>
                  - Feature 6
                </label>
              </span>
            </span>
          </button>
        </div>
        <input
          name="profileType"
          type="hidden"
          value="none"
        />
      </div>
      <div
        class="form-group"
        style={
          Object {
            "marginTop": "8%",
          }
        }
      >
        <button
          class="btn btn-primary nurseryApplicationButtons"
          style={
            Object {
              "backgroundColor": "rgb(157,0,0)",
              "margin": 0,
              "marginLeft": "7.5%",
              "marginRight": "2.5%",
              "width": "40%",
            }
          }
          type="button"
        >
          Cancel
        </button>
        <button
          class="btn btn-primary"
          style={
            Object {
              "backgroundColor": "rgb(1,106,1)",
              "margin": 0,
              "marginLeft": "2.5%",
              "marginRight": "7.5%",
              "width": "40%",
            }
          }
          type="submit"
        >
          Register
        </button>
      </div>
    </form>
  </div>
</div>
`;
